doc/api/os.md

```diff
@@ -246,5 +252,622 @@ operating system. This differs from the result of `os.homedir()`, which queries
 several environment variables for the home directory before falling back to the
 operating system response.
 
+## OS Constants
+
+The following constants are exported by `os.constants`. **Note:** Not all
+constants will be available on every operating system.
+
+### Signal Constants
+
+The following signal constants are exported by `os.constants.signals`:
+
+<table>
+<tr>
+<th>Constant</th>
+<th>Description</th>
+</tr>
+<tr>
+<td><code>SIGHUP</code></td>
+<td>Sent to indicate when a controlling terminal or process is closed.</td>
```

Comment:
nit: This is gramatically ambiguous, I’d prefer something like “when a controlling terminal is closed or a parent process exits” (btw, I am aware that most of these are just taken from manpages etc., you don’t have to change any of these for me… ;) )